resources/translations/en.edn

```diff
@@ -218,36 +218,42 @@
+          :application-returned {:subject "Application has been returned for modifications (%3)"
+                                 :message-to-applicant "Dear %1,\n\nYour application %3 has been returned to you for modifications.\n\nYou can modify and re-submit the application: %6"
+                                 :message-to-handler "Dear %1,\n\n%2 has returned the application %3 for modifications to the applicant %4.\n\nView application: %6"}
+          :application-licenses-added {:subject "New terms of use waiting for approval (%3)"
+                                       :message "Dear %1,\n\n%2 has requested your acceptance for new terms of use for application %3.\n\nYou can view the application and accept the terms of use: %6"}
+          :application-resubmitted {:subject "Application has been re-submitted (%3)"
+                                    :message "Dear %1,\n\nApplication %3 has been re-submitted by %4.\n\nYou can view the application: %6"}
+          :application-submitted {:subject "A new application has been submitted (%3)"
+                                  :message "Dear %1,\n\n%4 has submitted a new application %3 to access resource(s) %5.\n\nYou can view the application: %6"}
+          :comment-requested {:subject "Review request (%3)"
+                              :message "Dear %1,\n\n%2 has requested your review on application %3 submitted by %4.\n\nYou can review the application: %6"}
+          :commented {:subject "Application has been reviewed (%3)"
+                     :message "Dear %1,\n\n%2 has reviewed the application %3 submitted by %4.\n\nYou can view the application and the review: %6"}
+          :decided {:subject "Decision notification (%3)"
+                    :message "Dear %1,\n\n%2 has sent a decision on application %3 submitted by %4.\n\nYou can view the application and the decision: %6"}
+          :decision-requested {:subject "Decision requested (%3)"
+                               :message "Dear %1,\n\n%2 has requested your decision on application %3 submitted by %4.\n\nYou can review application: %6"}
+          :footer "\n\nPlease do not reply to this automatically generated message."
           :member-added {:subject "Added as a member of an application (%3)"
                          :message "Dear %1,\n\nYou've been added as a member of application %3.\n\nView application: %6"}
           :member-invited {:subject "Invitation to participate in an application"
-                           :message "Hello,\n\nThis email address (%1) has been invited to participate in an application.\n\nParticipate: %2"}}
+                           :message "Dear %1,\n\nYou have been invited to participate in an application submitted by %2. The title of the application is %3.\n\nYou can view the application and accept the terms of use: %4"}
+          :remarked {:subject "Application has been commented (%3)"
```

Comment:
Hmm, I guess this should be "Application has been commented on"

Reply:
done